🐙 source-scryfall: run up-to-date pipeline [2024-12-11]

airbyteio opened the pull request:
# Update source-scryfall

This PR was autogenerated by running `airbyte-ci connectors --name=source-scryfall up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- PATCH bump source-scryfall version to 0.0.4: Successful

- Build source-scryfall docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=49093) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=49093) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=49093) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
PATCH bump source-scryfall version to 0.0.4
Build source-scryfall docker image for platform(s) linux/amd64, linux/arm64
Create or update pull request on Airbyte repository
Add changelog entry
